streamlit_authenticator/authenticate.py

```diff
@@ -191,7 +265,7 @@ def login(self, form_name: str, location: str='main') -> tuple:
                 if login_form.form_submit_button('Login'):
                     self._check_credentials()
 
-        return st.session_state['name'], st.session_state['authentication_status'], st.session_state['username']
+        return st.session_state.get("name", ""), st.session_state.get('authentication_status', ""), st.session_state.get('username', "")
```

Comment:
Can you please elaborate on why exactly you have changed this?

Reply:
If the key doesnt exist, get with set the default value to an empty string.